src/relay/backend/graph_runtime_codegen.cc

```diff
@@ -56,7 +56,7 @@ struct LoweredOutput {
   std::string graph_json;
   Map<String, IRModule> lowered_funcs;
   Array<tvm::runtime::Module> external_mods;
-  std::unordered_map<std::string, tvm::runtime::NDArray> params;
+  std::unordered_map<std::string, std::pair<int, const tvm::runtime::NDArray>> params;
```

Comment:
This change will affect UpdateConstants() in line 379. Either we need to modify the UpdateConstants function or have something similiar to param_storage_ids in LoweredOutput separately